pkg/kubevirt/types.go

```diff
@@ -31,6 +31,8 @@ const (
 	MachineControllerManagerMonitoringConfigName = "machine-controller-manager-monitoring-config"
 	// MachineControllerManagerVpaName is the name of the VerticalPodAutoscaler of the machine-controller-manager deployment.
 	MachineControllerManagerVpaName = "machine-controller-manager-vpa"
+	// CloudProviderConfig is the name of the configmap containing the cloud provider config.
+	CloudProviderConfig = "cloud-provider-config"
```

Comment:
I don't think you need this configmap for anything.

Reply:
comment revised, it is the secret data key.